functions/sum.h

```diff
@@ -0,0 +1,29 @@
+#ifndef FUNCTIONS_SUM_H
+#define FUNCTIONS_SUM_H
+
+// Usage: SUM<A, B>
+// Adds A + B
+// A, B: INTEGER
+// return value: INTEGER
+
+class BladeBase;
+
+template<class A, class B>
```

Comment:
We should fix this template to be able to sum multiple things.
See LayerFunctions<> for now to do that.

Reply:
I modified using same approach as LayerFunctions to take multiple functions